docs/source/installation.rst

```diff
@@ -0,0 +1,146 @@
+Installation Guide
+************************
+
+Background Information
```

Comment:
I would put this info elsewhere, you could start a blog if you want or place this at the end. It's too focused on conda envs, regular python venvs are easier and faster if you're not using windows.

Reply:
Since the CaImAn docs also use conda envs, wouldn't it make sense to continue that pattern?

Reply:
There are people who use regular python venvs